refresh.lua

```diff
@@ -0,0 +1,19 @@
+-- refresh.lua: => Arguments: [value, ttl]
+-- refresh.lua refreshes provided keys's ttls if all their values match the input. 
+
+-- Check all keys values matches provided input.
+local values = redis.call("mget", unpack(KEYS))
+for i, _ in ipairs(KEYS) do
+	if values[i] ~= ARGV[1] then
+		return redis.error_reply("redislock: not obtained")
```

Comment:
that's NOT a good way to deal with errors

Reply:
Updated !